mlir/utils/jenkins/Jenkinsfile

```diff
@@ -723,7 +723,7 @@ pipeline {
                 axes {
                     axis {
                         name 'ARCH'
-                        values 'gfx908', 'gfx90a', 'gfx1030'
+                        values 'gfx908', 'gfx90a', 'gfx942'
```

Comment:
Can you kick weekly run on this branch and see if gfx942 is being picked and it runs successfully ?